src/swap/core/getTransaction.ts

```diff
@@ -0,0 +1,30 @@
+import { RawTransactionData, Transaction } from '../types';
+
+/**
+ * Constructs an unsigned transaction.
+ */
+export function getTransaction(tx: RawTransactionData, chainId: string): Transaction {
```

Comment:
We probably want to change the name of this one, as I read this more `get info from a particular transaction`. Which I think in this case is slighly different.